thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -294,6 +299,23 @@ export default Ember.Controller.extend({
   },
 
   /**
+   * If these two conditions are true, we assume the user wants to edit an existing alert group
+   * @method isSubmitDisabled
+   * @param {Number} metricId - Id of selected metric to graph
+   * @return {Boolean} PreventSubmit
+   */
+  isAlertGroupEditModeActive: Ember.computed(
+    'selectedConfigGroup',
+    'newConfigGroupName',
+    function() {
+      if (this.selectedConfigGroup && Ember.isNone(this.newConfigGroupName)) {
```

Comment:
it should be `this.get('selectedConfigGroup')` and  `this.get('newConfigGroupName')`. Otherwise, I believe it'll always be true because it returns a function (computed property)